Analysis/DataModel/include/AnalysisDataModel/HFCandidateSelectionTables.h

```diff
@@ -25,9 +25,10 @@ namespace o2::aod
 {
 namespace hf_selcandidate_lc
 {
-DECLARE_SOA_COLUMN(IsSelLc, isSelLc, int);
+DECLARE_SOA_COLUMN(IsSelLcpKpi, isSelLcpKpi, int);
+DECLARE_SOA_COLUMN(IsSelLcpiKp, isSelLcpiKp, int);
```

Comment:
Is there a reason why Anton's suggestion about using `int8_t` has not been addressed?

Reply:
I am not sure @vkucera

Reply:
@vkucera . I tried but O2 does not compile with int8_t

Reply:
Could you please show me the log? `int8_t` should be supported as a column type. One can also try with `uint8_t`.

Reply:
There is no reason why O2 should not compile because of an `int8_t` column. It is already used in the `FlagMCMatchRec` and `FlagMCMatchGen` columns.

Reply:
I'm sorry, I misremembered. O2 compile correctly but I get this error when I run the validation task:

[ERROR] SEVERE: Device hf-task-lc (8240) had at least one message above severity ERROR: Exception caught: Failed to create filter: ExpressionValidationError: Function bool greater_than_or_equal_to(int8, int32) not supported yet.

Reply:
I see. That indeed looks like an O2 problem in the comparison between `int` and `int8_t`.

Reply:
When declaring a filter one can explicitly cast the value to `int8_t` in order to avoid this. For example `Filter f = isSelLcpKpi == (int8_t)1;`